README.md

```diff
@@ -174,6 +180,13 @@ all cases, but for the 2NT bids we use Puppet Stayman.
 * 3♣️! Natural, signoff
 * 3♦️! Invitation to 3NT. Requires long diamonds (6+) as opener passes with
   minimum.
+* 3♥️ Invite to 4♥️, guarantees 5+♥️s
```

Comment:
Invite with 10-12?

Reply:
I actually prefer not to define what invitational strength means. This is just a bid that sets trump as hearts and asks opener to pass with near the bottom of their range and bid 4H with near the top of their range. 10-12 should usually fall into that category, but potentially other hands can as well.

Reply:
Discussed offline. Use judgment.